subworkflows/local/interaction_caller/hicexplorer.nf

```diff
@@ -0,0 +1,23 @@
+/*
+ * Call interaction peaks by HiCExplorer, current not proper
+ */
+
+include { HICEXPLORER_CHICSIGNIFICANTINTERACTIONS      } from '../../../modules/local/hicexplorer/chicsignificantinteractions'
+
+
+workflow HICEXPLORER_INTERACTIONS {
+    take:
+    matrix                          // [ bin_size, [cool] ] corrected matrix file
+
+    main:
+    ch_multiqc_files = Channel.empty()
```

Comment:
Not sure I follow the logic with this `ch_multiqc_files`>
Can you explain the idea behind that?

Reply:
I leave it as standard output for all downstream analysis. Sorry I don't have time to implement them yet but they in TODO list.

Reply:
This will be fixed during the reviewer process.